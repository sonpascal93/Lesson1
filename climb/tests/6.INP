200
5 -14 -13 10 -9 0 -15 -13 -3 2 -1 2 14 16 -15 -19 -19 -15 0 -12 -9 -14 11 11 6 5 12 -5 -8 0 -4 0 16 10 10 3 -6 -7 -3 -17 6 -16 -11 -6 -13 19 -18 16 19 -13 -12 3 -19 -19 5 -14 13 5 18 -9 15 17 -17 -17 -4 -5 5 -18 8 5 -4 4 15 2 8 13 1 2 -18 -17 -11 -10 2 15 -1 18 9 -1 -11 17 -5 -4 4 18 -8 -17 -5 -19 -9 -17 3 -10 -10 6 17 -12 1 -8 15 -4 0 5 -1 10 8 5 -5 17 -13 7 -17 7 1 2 2 -2 0 -1 -16 -13 7 15 19 3 18 -3 7 8 -8 -18 13 -2 14 -8 -5 -17 12 15 -2 -11 -8 12 4 -2 -10 15 -13 -9 -14 -4 16 -2 -9 -12 -1 -7 -3 16 8 4 0 3 -5 3 -7 11 1 -16 -2 8 -8 -5 15 0 5 -15 0 0 18 -19 -13 6 9 2 -17 -9 -19 -10 5 0 