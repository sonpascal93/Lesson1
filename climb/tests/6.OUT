40
16 17 18 20 21 29 31 32 36 52 55 58 67 70 75 76 84 86 90 105 109 114 115 120 122 124 125 127 128 142 149 154 162 167 173 175 181 186 190 197 